File: backend/src/main/java/ar/edu/unq/reclamar/utils/Localizacion.java
package ar.edu.unq.reclamar.utils;

import javax.persistence.Entity;

import org.springframework.data.jpa.domain.AbstractPersistable;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@Entity
@JsonIgnoreProperties(value = {"new"})
public class Localizacion  extends AbstractPersistable<Long> {
	
	private static final long serialVersionUID = 1L;
	
	String latitud;
	String longitud;
	String direccionFisica; 
	
	public String getLatitud() {
		return latitud;
	}
	public void setLatitud(String latitud) {
		this.latitud = latitud;
	}
	public String getLongitud() {
		return longitud;
	}
	public void setLongitud(String longitud) {
		this.longitud = longitud;
	}
	public String getDireccionFisica() { 
		return direccionFisica; 
	} 
	public void setDireccionFisica(String direccionFisica) { 
		this.direccionFisica = direccionFisica; 
	} 
}

File: frontend/src/app/model/localizacion.ts
export class Localizacion {
    public latitud:string;
    public longitud:string;
    public direccionFisica:string;

    public set setLatitud(lat:string){
        this.latitud = lat
    }

   public set setLongitud(long:string){
        this.longitud = long
    }

    public set setDireccionFisica(dir:string){
        this.direccionFisica = dir
    }

    constructor(lat:string, lng:string, dirFisica: string) {
        this.latitud=lat;
        this.longitud=lng;
        this.direccionFisica = dirFisica;
    }

    static crearDesdeJson(json:any): Localizacion{
     const localizacion = new Localizacion(json.latitud, json.longitud, json.direccionFisica)
      return localizacion
    }
}
